docker/app/REVISION

```diff
@@ -0,0 +1 @@
+d642734d6d5be5847222e429130cf2e586c81ee1
```

Comment:
Should this have been committed?

Reply:
The seems to be working fine, system_status/details is reflecting different revisions for each deployment, so probably doesn't matter.

Reply:
It just needs to exist so we could change the content to make it clearer it's just for testing

Reply:
Thank you!